README.md

```diff
@@ -73,34 +72,11 @@ Once the project is configured, use `ninja` to build it.
 To install it into the default prefix, use `ninja install`.
 If you want to specify custom `DESTDIR`, use `DESTDIR=... ninja install`.
 
-## Building the Debian Package
-
-To build a .deb file for installation on Debian platforms, perform the
-following steps:
-+ Run `sudo apt install devscripts` to install the `debuild` utility.
-+ Update the `debian/changelog` to reflect the changes in this release.
-+ If necessary, update the year of the copyright in `debian/copyright`.
-+ Clean any build artifacts: Run `make clean` and delete the `bin` directory,
-if it exists.
-+ Run `tar -xcvf ../urbit-x.y.z.orig.tar.gz .` from the top-level folder in
-the repo.  This command will create an archive in the directory above the
-current directory, which will be used in packaging.
-+ Run `debuild -us -uc`, also from the top-level folder in the repo. This
-creates a .deb file in the folder above the current directory.
-
-The resulting .deb file should now exist in the folder above the current
-directory. To test that the .deb file works properly, you can perform the
-following steps:
-+ Uninstall urbit: `sudo apt remove urbit`.
-+ Run `sudo dpkg -i ../urbit-x.y.z_amd64.deb` to install the new version.
-+ Boot up a ship using the `urbit` command.
-
 ## Contact
 
 If you have any questions, problems, patches, or proposals for patches, please
 feel free to get in touch in whatever way is most convenient:
 
-- Post to `/urbit-meta` on Urbit `:talk`.  (You can do this via
-[urbit.org/stream](https://urbit.org/stream) without a running Urbit).
+- Post to `/urbit-meta` on Urbit [`:talk`](https://urbit.org/docs/learn/arvo/arvo-internals/messaging/).
```

Comment:
This new link is not currently correct, but I believe it will be shortly.

Reply:
@msutherl are we changing the "help" channel to /urbit-help?